lib/util/import-file.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * imports 'mjs', else requires.
+ * NOTE: require me late!
+ * @param {string} filepath
+ * @todo WARN on version 10 and '--experimental-modules' and '--esm'
+ */
+module.exports = function importFile(filepath) {
+  return filepath.endsWith('.mjs')
+    ? import(require('url').pathToFileURL(filepath))
```

Comment:
do we need to lazy-load `url` as well here?

Reply:
Not really , it can be a top level import. I'm maybe a bit paranoid there
I guess I test run can decide on it.

Reply:
Run a few tests with the top level `require` 
And i get some test failures, ... timeout related 
It could be just my environment 
But, I think it could  be making the initialization a bit more expensive. 
Perhaps its better to leave it 'lazy' , and let only the `--esm` users pay the minimal performance penalty 
As Its cached by require anyway

These are the failures
```

  1) Integration Tests
       sqlite3 | sqlite3
         Updates
           should immediately return updated value for other connections when updating row to DB returns:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (.../knex/test/all-tests-suite.js)
      at listOnTimeout (internal/timers.js:551:17)
      at processTimers (internal/timers.js:494:7)

  2) Integration Tests
       sqlite3 | sqlite3
         Updates
           should increment a float value:
     AssertionError: expected 676.53 to be close to 0 +/- 0.001
      at .../knex/test/integration/builder/updates.js:185:57

  3) Integration Tests
       sqlite3 | sqlite3
         Updates
           should allow returning for updates in postgresql:
     KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?
      at Client_SQLite3.acquireConnection (lib/client.js:349:26)
```